features/step_definitions/tag_steps.rb

```diff
@@ -443,3 +444,7 @@
   tag = Tag.find_by(name: tagname)
   puts tag.inspect
 end
+
+Then "no tag is scheduled for count update from now on" do
+  Tag.any_instance.should_not_receive(:write_taggings_to_redis)
```

Comment:
You can use the expect syntax here, like: https://github.com/otwcode/otwarchive/blob/cb58df0bc28bcddc6b07f485d9d981b043ae1397/spec/controllers/troubleshooting_controller_spec.rb#L123